plan/find_best_task.go

```diff
@@ -29,7 +29,7 @@ const (
 	netWorkFactor      = 1.5
 	netWorkStartFactor = 20.0
 	scanFactor         = 2.0
-	descScanFactor     = 5 * scanFactor
+	descScanFactor     = 2 * scanFactor
```

Comment:
Please add comments here to explain why desc-scan is two times larger than scan?